engine/interfaces/src/main/scala/org/trustedanalytics/atk/domain/DomainJsonProtocol.scala

```diff
@@ -562,5 +562,6 @@ object DomainJsonProtocol extends AtkDefaultJsonProtocol with EventLogging {
 
   implicit val hBaseArgsSchemaFormat: RootJsonFormat[HBaseSchemaArgs] = jsonFormat3(HBaseSchemaArgs)
```

Comment:
Just for clarity. Could we remove the RootJsonFormat[<>]. It is implicit that we define jsonSchema and need not say the type explicitly. If you see the file, look at how the items above this are defined.